rpc_jobs/setup_statsd.yml

```diff
@@ -0,0 +1,129 @@
+# Merge Trigger job for setup statsd.
+# This used to redeploy whenever there are merges to rpc-gating.
+- job:
+    name: "Merge-Trigger-StatsD"
+    project-type: freestyle
+    scm:
+      - git:
+          url: https://github.com/rcbops/rpc-gating
+          branches:
+           - master
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    triggers:
+        - github
+    builders:
+      - trigger-builds:
+        - project:
+            - "Setup-StatsD"
+
+- job:
+    name: Setup-StatsD
+    project-type: pipeline
+    concurrent: false
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    parameters:
+      - rpc_gating_params
+    triggers:
+      - timed: "@daily"
```

Comment:
Is this necessary if we already have a push trigger for this?

Reply:
The daily execution ensures that even if there are no merges going in, the playbook still executes. This ensures that packages are updated and the environment remains configured with the latest keys and such.

Reply:
Fair. :)